utils/gen-vote-contents.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+VERSION=$1
+
+SUBSTRING1=$(echo $VERSION| cut -d'.' -f 1)
+SUBSTRING2=$(echo $VERSION| cut -d'.' -f 2)
+BLOB_VERSION=$SUBSTRING1.$SUBSTRING2
+
+read -p "Please enter release note pr: " RELEASE_NOTE_PR
+read -p "Please enter release commit id: " COMMIT_ID
```

Comment:
The COMMIT_ID is the latest commit, which can be fetched automatically via git cmd?